server

```diff
@@ -383,7 +383,7 @@ amzn_install() {
 
 check_centos_version() {
   case "$VERSION_ID" in
-    "7"|8.?)
+    "7"|"8"|8.?)
```

Comment:
what about "8.*" or a similar wildcard pattern?

Reply:
yes, added.

Reply:
The commit message also updated

Reply:
I meant something like `7* | 8*` which should catch everything under 7,8 (include)

Reply:
Changed.